src/bundles/page-template.js

```diff
@@ -68,58 +71,74 @@ function PageTemplate({ children, fullScreen, headerActions, alerts }) {
       href: urls.getHomeUrl(),
       text: t('pageNames:home'),
       icon: HomeOutlined,
-      permission: null
+      permission: null,
+      show: true
+    },
+    {
+      key: 'my-space',
+      href: urls.getMySpaceUrl(),
+      text: t('pageNames:mySpace'),
+      icon: IdcardOutlined,
+      permission: null,
+      showWhen: !!user
     },
     {
       key: 'docs',
       href: urls.getDocsUrl(),
       text: t('pageNames:docs'),
       icon: FileOutlined,
-      permission: permissions.VIEW_DOCS
+      permission: permissions.VIEW_DOCS,
+      showWhen: true
     },
     {
       key: 'users',
       href: urls.getUsersUrl(),
       text: t('pageNames:users'),
       icon: UserOutlined,
-      permission: permissions.EDIT_USERS
+      permission: permissions.EDIT_USERS,
+      showWhen: true
     },
     {
       key: 'settings',
       href: urls.getSettingsUrl(),
       text: t('pageNames:settings'),
       icon: SettingOutlined,
-      permission: permissions.EDIT_SETTINGS
-    }
-  ];
-
-  if (!clientConfig.disabledFeatures.includes(FEATURE_TOGGLES.import)) {
-    pageMenuItems.push({
+      permission: permissions.EDIT_SETTINGS,
+      showWhen: true
+    },
+    {
       key: 'import',
       href: urls.getImportsUrl(),
       text: t('pageNames:importBatches'),
       icon: ImportOutlined,
-      permission: permissions.MANAGE_IMPORT
-    });
-  }
-
-  if (settings?.helpPage?.[language]) {
-    pageMenuItems.push({
+      permission: permissions.MANAGE_IMPORT,
+      showWhen: !clientConfig.disabledFeatures.includes(FEATURE_TOGGLES.import)
+    },
+    {
       key: 'help',
-      href: urls.getDocUrl(settings.helpPage[language].documentKey, settings.helpPage[language].documentSlug),
-      text: settings.helpPage[language].linkTitle,
+      href: helpPage ? urls.getDocUrl(helpPage.documentKey, helpPage.documentSlug) : '',
+      text: helpPage?.linkTitle,
       icon: QuestionOutlined,
-      permission: permissions.EDIT_SETTINGS
-    });
-  }
-
-  pageMenuItems.push({
-    key: 'language',
-    onClick: () => setIsUiLanguageDialogVisible(true),
-    text: t('common:language'),
-    icon: GlobalOutlined,
-    permission: null
-  });
+      permission: permissions.EDIT_SETTINGS,
```

Comment:
Should be `null` for the help menu item.

Reply:
Made it `null`, thank!

Reply:
:+1: